Lib/idlelib/run.py

```diff
@@ -229,7 +229,7 @@ def get_message_lines(typ, exc, tb):
             sys.__excepthook__(typ, exc, tb)
         return [err.getvalue().split("\n")[-2] + "\n"]
     else:
-        return traceback.format_exception_only(typ, exc)
+        return traceback.format_exception_only(exc)
```

Comment:
Please omit this until after 3.10 end-of-maintenance, next May.

Reply:
These aliases are valid in 3.10. This shouldn’t cause issues with backports.

Reply:
Are you planning to backport (no label yet).  Otherwise, a separate PR would be easier.

Reply:
I’m not planning to backport, but I’m avoiding changes that can complicate future backports (so I’m holding off on whatever depends on 3.11).